python/dgl/sparse/softmax.py

```diff
@@ -8,18 +8,13 @@
 __all__ = ["softmax"]
 
 
-def softmax(A: SparseMatrix) -> SparseMatrix:
-    """Apply row-wise softmax to the non-zero entries of the sparse matrix.
+def softmax(input: SparseMatrix) -> SparseMatrix:
```

Comment:
Are we making the change for all usage of `A` across APIs?

Reply:
Yes, for APIs on a single SparseMatrix and link to the class method, we use input.